climada/__init__.py

```diff
@@ -93,10 +91,4 @@ def setup_climada_data(reload=False):
                 src = Path(__file__).parent.parent.joinpath(src_dir, path.name)
                 copyfile(src, path)
 
-
-def init():
-    setup_climada_data()
-    setup_logging(CONFIG.log_level.str())
-
-
-init()
+setup_climada_data()
```

Comment:
The `init` function is not needed after https://github.com/CLIMADA-project/climada_petals/pull/3 is settled.